server.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const path = require('path');
+const app = express();
+const PORT = 3000;
```

Comment:
```suggestion
const PORT = 3000;
const port = process.env.PORT || PORT;
```

Так будет возможность прокинуть порт снаружи